uvicorn/config.py

```diff
@@ -368,12 +389,11 @@ def load(self):
 
         self.loaded = True
 
-    def setup_event_loop(self):
-        loop_setup = import_from_string(LOOP_SETUPS[self.loop])
-        if loop_setup is not None:
-            loop_setup()
+    def setup_event_loop(self) -> None:
+        loop_setup: Callable = import_from_string(LOOP_SETUPS[self.loop])
+        loop_setup()
```

Comment:
why the `if loop_setup is not None:` disappears ?

Reply:
Restored (d93bf88)